### ISSUE TITLE
spaced-line-comment fails sometimes with shebang

### ISSUE THREAD

**Issue by julien-f** [eslint/eslint#2368]
In the following example, I get *Expected space or tab after // in comment* at `1:0`:

```json
{
  "rules": {
    "spaced-line-comment": [2, "always"]
  }
}
```

```javascript
#!/usr/bin/env node
''
// Foo
''
```

I do not know if my test are relevant but the error disappears if:

- I delete any of the 3 following lines
- the second line is not a literal
- the third line is not a comment

**Comment by nzakas** [comment]
Might have something to do with how we remove the shebang. Will take a look.

**Comment by jrvidal** [comment]
It looks like a bug in espree. The parsed AST includes the shebang comment in the expected places _plus_ a `leadingComment` entry for the second string literal (!?).

[AST](https://gist.github.com/anonymous/5935938e458ca387e961)

**Comment by lo1tuma** [comment]
@jrvidal The shebang comment gets removed from the AST (see: [eslint.js](https://github.com/eslint/eslint/blob/3517fa03ab7c4aceb5909b0259b074aeb20b45b9/lib/eslint.js#L705-L712)).

**Comment by nzakas** [comment]
Yeah, looks like it's attaching that comment twice, once for each node. Can you for an Espree issue?

**Comment by nzakas** [comment]
@lo1tuma in this case the comment is being added twice, and we are just removing the first.

**Comment by platinumazure** [comment]
@nzakas Is this still an issue, or have we fixed this with all the changes to espree?

**Comment by nzakas** [comment]
I don't know, can you check? Heres the Espree issue: https://github.com/eslint/espree/issues/136

**Comment by kaicataldo** [comment]
I can take a look at this - feel pretty comfortable with the way we add comments in Espree now :)

**Comment by kaicataldo** [comment]
This appears fixed to me! My comment in the Espree issue: https://github.com/eslint/espree/issues/136#issuecomment-198778281

Just to be extra sure, just ran the latest version of ESLint on the JS in the original post with the following config and saw no errors.

```js
{
  "rules": {
    "spaced-comment": ["error", "always"]
  }
}
```

**Comment by ilyavolodin** [comment]
Nice. Closing as it looks like it's fixed now.
